### PR TITLE
Resolve 2 players zoning in within 500ms and not rendering the other

### LINKED ISSUE #5519: 🐛 Players don't always render for other players on zone-in
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## Branch affected by issue 

`base`

## Steps to reproduce

takes some work to get it to happen, but basically both players have to zone in at approximately the same time:
image

https://imgur.com/Uo7sGID
Notice the minimap showing no player as well, mowford2 never changed positions during the running back and forth

### PR DESCRIPTION
**Pull request by MowFord**
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?

Closes https://github.com/LandSandBoat/server/issues/5519 .

The original find i made in the issue seemed inefficient packet-wise, but this workaround should work just fine.

The described issue with leaving a trial by avatar fight makes sense as all players get zoned at the same time, so very likely to trigger the condition described in the issue

## Steps to test these changes

see https://github.com/LandSandBoat/server/issues/5519

### CODE REVIEW

**Comment by WinterSolstice8** [comment]
This works but it's not the right fix, the root cause appears to be that the client isn't yet ready to receive spawn packets. I need to do some quick caps to verify when the server starts to send spawn packets, but i should have something better tonight (assuming I don't die from having worked too much this week already)

Basically, my assumption is that 0x01A with the 0x14 action is when the client is ready for some kinds of packets, some info via atom0s (his name is cut off due to my crappy screen res, sorry atom0s):

image

This doesn't constitute what the server does

tl;dr: need to see if 0x01A send corresponds to SPAWN 0x00D/0x00Es happening after it's sent.  Then we need a latch to prevent zone entities from being spawned in the loop until the server has received that

**Comment by MowFord** [comment]
Cool deal, Ill just close this one out. We applied it as a workaround for now

**Comment by WinterSolstice8** [comment]
Yeah it's not a bad workaround for sure, it will get the job done while I work on this
